src/lib/components/card/Card.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
```

Comment:
Created a reusable card component so we can use for the entire grid. Need to flesh out for others